tgui/packages/tgui-panel/ping/middleware.js

```diff
@@ -32,9 +33,15 @@ export const pingMiddleware = store => {
     const { type, payload } = action;
     if (!initialized) {
       initialized = true;
-      setInterval(sendPing, PING_INTERVAL);
+      interval = setInterval(sendPing, PING_INTERVAL);
       sendPing();
     }
+    if (type === 'roundrestart') {
+      // Stop pinging because dreamseeker is currently reconnecting.
+      // Topic calls in the middle of reconnect will crash the connection.
+      clearInterval(interval);
+      return next(action);
+    }
     if (type === 'pingReply') {
```

Comment:
wait make this an else if

Reply:
No, because initial message should always fall through, regardless of what that message was.

Reply:
ok